src/executables/uber.js

```diff
@@ -5,6 +5,8 @@ import program from 'commander';
 import pkg from '../../package.json';
 
 program.version(pkg.version)
+  .description('Figure out if you should order a car to pick you up and drive you to where you want to go')
   .command('price', 'get price estimate')
-  .command('time', 'get time to pickup estimate')
+  .command('time [address]', 'get time to pickup estimate')
```

Comment:
It doesn't really work as an optional param because of how Commander is setup for github style commands. It doesn't change anything here.